cpp/include/cudf/fixed_point/temporary.hpp

```diff
@@ -61,18 +61,6 @@ constexpr auto abs(T value)
   return value >= 0 ? value : -value;
 }
 
-template <typename T>
-CUDF_HOST_DEVICE inline auto min(T lhs, T rhs)
-{
-  return lhs < rhs ? lhs : rhs;
```

Comment:
Just for the record -- this implementation was not a standards-compliant implementation for float/double values. https://godbolt.org/z/6P9h11YEn

> [Returns the first argument when the arguments are equivalent.](https://eel.is/c++draft/alg.min.max#2.sentence-2)
> -- alg.min.max